src/test/java/test/SimpleBaseTest.java

```diff
@@ -345,7 +346,21 @@ protected static File createDirInTempDir(String dir) {
    */
   protected static List<Integer> grep(File fileName, String regexp, List<String> resultLines) {
     List<Integer> resultLineNumbers = new ArrayList<>();
-    try (BufferedReader fr = new BufferedReader(new FileReader(fileName))) {
+    Reader reader = null;
+    try {
+      reader = new FileReader(fileName);
+      resultLineNumbers =  grep(reader, regexp, resultLines);
+    }catch (IOException e) {
+      e.printStackTrace();
+    } finally {
+      closeQuietly(reader);
```

Comment:
We use Java7, so you should use the try-with-resources feature.

Reply:
If it is intentional, just add a comment which explains why

Reply:
I will fix this. The `closeQuietly()` call should have been happening in the `BufferedReader`.

Reply:
@juherr - I have amended the commit and addressed this.